object.js

```diff
@@ -61,15 +61,17 @@ const tonicHTML = `
           <span><i class="fa fa-circle"></i></span>
           <span class="year">${tonic.year}</span>
         </div>
-        <p class="primarytext">${tonic.description}</p> 
+        <img src="${tonic.image}" alt="${tonic.name}">
+        <p id='adj' class="primarytext">${tonic.description}</p> 
       </div>
       <ul class="tags">
         <li class="tag">${tonic.technologies[0]}</li>
         <li class="tag">${tonic.technologies[1]}</li>
         <li class="tag">${tonic.technologies[2]}</li>
```

Comment:
- [ ] Nice efforts so far 👏 . However, let's make a final change to fit the template given.
- On the desktop, the `buttons` and technologies should be on the right.
- The aim is to stick to user requirements, and also give a user a better experience 👍

## View:
![image](https://user-images.githubusercontent.com/50990925/224525625-0b3f4f62-6269-40f8-a604-74876cfdf5d1.png)


## FIGMAs
![image](https://user-images.githubusercontent.com/50990925/224525644-d62bb0b9-11e7-4f29-8ca0-fbe838074bba.png)